Comp2024/src/main/java/frc/robot/subsystems/HID.java

```diff
@@ -37,6 +41,8 @@ public HID(GenericHID driver, GenericHID operator)
 
     m_driver = driver;
     m_operator = operator;
+    m_driverRumbleOn = driverRumbleOn;

```

Comment:
These two lines of code to initialize the rumble states aren't needed if they are initialized when declared as class member variables.

We're not really passing in rumble requests when the HID subsystem is created in robotContainer. I hope that explanation helps a bit.